de.dentrassi.pm.aspect.mvnosgi/OSGI-INF/aspect.properties

```diff
@@ -0,0 +1,4 @@
+drone.aspect.id=mvnosgi
+drone.aspect.name=Maven OSGi Channel Aspect
+drone.aspect.version=1.0.0
+drone.aspect.requires=osgi mvn
```

Comment:
I was wrong ... of course you need one part o the "mvn" aspect. The problem is you can only have both at the moment. Extrating the POM and extracting the meta data.